templates/weavescope/0/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+weavescope-probe:
+  image: weaveworks/scope:0.11.1
+  privileged: true
+  net: host
+  pid: host
+  labels:
+    io.rancher.scheduler.global: true
+    io.rancher.container.dns: true
+  links:
```

Comment:
@cloudnautique can you test it without the link? ...I'd expect it to work without it.